src/server/pegasus_server_impl.cpp

```diff
@@ -675,12 +697,22 @@ void pegasus_server_impl::on_multi_get(const ::dsn::apps::multi_get_request &req
         return;
     }
 
-    int32_t max_kv_count = request.max_kv_count > 0 ? request.max_kv_count : INT_MAX;
+    uint32_t max_kv_count = request.max_kv_count > 0 ? request.max_kv_count : INT_MAX;
+    uint32_t max_iteration_count = std::min(max_kv_count, _rocksdb_max_iteration_count);
```

Comment:
After the PR, we should update client doc, the returned kv count/size maybe less than user requested.

Reply:
Sure, I will update related document~